arch/arm/core/cortex_m/tz/arm_core_tz.c

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2018 Nordic Semiconductor ASA.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <arch/arm/cortex_m/cmsis.h>
+#include "cortex_m/tz.h"
```

Comment:
Should only use \" includes for relative paths.

Reply:
Thanks, will fix.